### ISSUE TITLE
Account growth inputs are static -  should change dynamically

### ISSUE THREAD

**Issue by christophertull** [California-Data-Collaborative/RateComparison#47]
@monobina ran across an error that arises [here](https://github.com/California-Data-Collaborative/RateComparison/blob/006f4a7cbcdfb25a7e373f622db9b6a21508ffd3/server.R#L60)

```R
#for generating customer class
class_proportions <- as.data.frame(prop.table(table(df$cust_class)), stringsAsFactors = FALSE)
      
class_proportions$Freq <- c(input$Commercial,input$Institutional, input$Irrigation,input$Other,input$ResidentialMulti, input$ResidentialSingle)
```

Note how the `class_proportions` are generated dynamically from the dataframe, but the `input`s are hardcoded. This throws an error if there are more (or fewer?) classes in the dataframe than there are inputs. To get around this we should probably generate the account growth inputs in a loop.

The error for reference:

```R
Stack trace (innermost first):
    84: $<-.data.frame
    83: $<- [C:\Users\mmukherjee\Desktop\MNWD\RateRP3.2.9/server.R#62]
    82: <reactive:planneddf> [C:\Users\mmukherjee\Desktop\MNWD\RateRP3.2.9/server.R#62]
    71: planneddf
    70: <reactive:DF> [C:\Users\mmukherjee\Desktop\MNWD\RateRP3.2.9/server.R#511]
    59: df_original
    58: dropNulls
    57: updateSliderInput
    56: observerFunc [R/class_graphs.R#223]
     1: runApp
Warning: Error in $<-.data.frame: replacement has 6 rows, data has 7
```

**Comment by monobina** [comment]
@christophertull  yes I fixed this in my local code by adding the Recycled input. Is this originally written by @anudeepvanjavakam1 ? one of us can create the loop

**Comment by christophertull** [comment]
Just a note that this will also need to affect the calls to `req(input$ResidentialSingle)` etc.

**Comment by christophertull** [comment]
Closed by #44
